northstar/src/api/model.rs

```diff
@@ -141,6 +141,15 @@ pub enum Request {
         container: Container,
         signal: i32,
     },
+    /// Start a new process in a container
+    Exec {
+        /// Container
+        container: Container,
+        /// Command and arguments
+        cmd: Vec<NonNullString>,
```

Comment:
Should we add env variables?

Reply:
yes! added `-e,--env` flag for passing env variables :h